k8s/bases/swe-idcc/swe-idcc-runner-itools-configmap.yaml

```diff
@@ -28,35 +22,47 @@ data:
       plausibleActivePowerLimit: 10000
       maxIteration: 100
       newtonRaphsonConvEpsPerEq: 1.0E-3
-      minRealisticVoltage: 0.5
-      maxRealisticVoltage: 1.5
-
-    rao-parameters:
-      objective-function: MAX_MIN_MARGIN_IN_AMPERE
-      max-number-of-iterations: 5
-      optimization-solver: CBC
-      relative-mip-gap : 0.001
-      pst-optimization-approximation: APPROXIMATED_INTEGERS
+      minRealisticVoltage : 0.5
+      maxRealisticVoltage : 1.5
+    
+    rao-objective-function:
+      type: MAX_MIN_MARGIN_IN_AMPERE
+      preventive-stop-criterion: SECURE
+      curative-stop-criterion: SECURE
+    
+    rao-range-actions-optimization:
+      max-mip-iterations: 5
       pst-penalty-cost: 0.01
       pst-sensitivity-threshold : 0.0
-      sensitivity-fallback-overcost: 0.0
-      rao-with-loop-flow-limitation: false
-      rao-with-mnec-limitation: false
-      perimeters-in-parallel: 14
+      pst-model: APPROXIMATED_INTEGERS
+    
+    rao-linear-optimization-solver:
+      solver: CBC
+      relative-mip-gap: 0.001
+    
+    rao-topological-actions-optimization:
+      max-search-tree-depth: 2
+      relative-minimum-impact-threshold: 0.0
+      absolute-minimum-impact-threshold: 1.0
+    
+    rao-second-preventive-rao:
+      execution-condition: POSSIBLE_CURATIVE_IMPROVEMENT
+      re-optimize-curative-range-actions: false
+      hint-from-first-preventive-rao: true
+    
+    rao-ra-usage-limits-per-contingency:
+      max-curative-ra: 10
+      max-curative-tso: 2
+      max-curative-ra-per-tso: [ "{RTE}:5" ]
+    
+    rao-not-optimized-cnecs:
+      do-not-optimize-curative-cnecs-for-tsos-without-cras: false
+    
+    rao-load-flow-and-sensitivity-computation:
       load-flow-provider: OpenLoadFlow
       sensitivity-provider: OpenLoadFlow
     
-    search-tree-rao-parameters:
-      maximum-search-depth: 2
-      preventive-rao-stop-criterion: SECURE
-      curative-rao-stop-criterion: SECURE
-      absolute-network-action-minimum-impact-threshold: 1.0
-      relative-network-action-minimum-impact-threshold: 0.0
-      preventive-leaves-in-parallel: 14
+    rao-multi-threading:
+      contingency-scenarios-in-parallel: 4
```

Comment:
Could you please update the contingerny-scenarios-in-parallel and the preventive-leaves-in-parallel with the previous values ?